src/GraphQL/ResolveFieldContext/IResolveConnectionContext.cs

```diff
@@ -1,20 +1,47 @@
+using GraphQL.Resolvers;
+
 namespace GraphQL.Builders
 {
+    /// <summary>
+    /// Contains parameters pertaining to the currently executing <see cref="IFieldResolver"/>, along
+    /// with helper properties for resolving forward and backward pagination requests on a
+    /// connection type.
+    /// </summary>
     public interface IResolveConnectionContext : IResolveFieldContext
     {
+        /// <summary>
+        /// Indicates if this connection only allows forward pagination requests.
+        /// </summary>
         bool IsUnidirectional { get; }
 
+        /// <summary>
+        /// For a forward pagination request, returns the maximum number of edges to be returned.
```

Comment:
I have to admit that I haven't read about relay yet, but this comment and one below for `Last` seem odd.

Reply:
I paraphrased from the spec. See “arguments” here:

https://relay.dev/graphql/connections.htm

Probably not the best description but that’s the best I could come up with.

Reply:
From the third reading, I roughly understood what it was about. Let's give each property and class a link to the relevant paragraphs of the specification.

Reply:
Perhaps I misunderstood you, but we don't have embedded links to the graphql specification anywhere else in the xml documentation.  The xml comments assume a basic understanding of the overall functionality, and are written as such (just like the rest of the xml comments).  Further, some of the properties have no bearing on the actual spec, and probably shouldn't even exist.  For instance, `IsUnidirectional` is only a placeholder to hold temporary data within the program.

I'm leaving it as-is for now.  If we need more overall documentation for relay types, we should add it to the https://graphql-dotnet.github.io/docs/getting-started/relay page.  (As the page is nearly empty right now!)  I did add a link to the spec on that page.

Reply:
OK. In general, sooner or later, I expected to remove all these relay types in favor of a separate relay project.